task_3.py

```diff
@@ -0,0 +1,21 @@
+"""Introduction to programming’: Task 3,
+   Kyshynska Taisiia"""
+import math as m
+
+print('Introduction to programming’: Task 2')
+print('Kyshynska Taisiia')
+
+TEMPLATE = '{} = '
+x = float(input(TEMPLATE.format('x')))
+z = float(input(TEMPLATE.format('z')))
+
+if isinstance(x, (int, float)) and isinstance(z, (int, float)): # для проверки того, что переменные x и z являются числами,
+    y = (x + m.exp(z - 1)) / (1 - (x ** 2) * abs(x - z)) #а затем продолжаем выполнение кода, если это так.
+
+    if (x**2) * abs(x - z) == 1:
```

Comment:
I haven't checked but I believe the code will fail at the line №13 if the values are out of scope, aren't they?